packages/protocol/blockchain_scripts/airdrop/2022-01-24-backers/calculation.ts

```diff
@@ -0,0 +1,97 @@
+import {promises as fs} from "fs"
+import {BigNumber as BigNum} from "bignumber.js"
+import {parseCsv} from "../community/parseCsv"
+import {program} from "commander"
+import {decimals} from "@goldfinch-eng/protocol/test/testHelpers"
+import {
+  LIQUIDITY_PROVIDER_GRANT_REASON,
+} from "@goldfinch-eng/protocol/blockchain_scripts/merkle/merkleDistributor/types"
+
+async function getNotVestedContracts() {}
+
+interface RawCSV {
+  poolName: string
+  poolAddress: string
+  backerAddress: string
+  usdcAmount: string
+  cappedAirdropGFI: string
+  uncappedAirdropGFI: string
+  riskBonusGFI: string
+  modeledLiqMiningGFI: string
+  totalGfiDistribution: string
+  immediatelyAvailable: string
+  unlocksOver12months: string
+  returnAtPrice: string
+}
+
+interface PreparedRow {
+  backerAddress: string
+  immediatelyAvailable: BigNum,
+  vesting: BigNum
+}
+
+function sanitizeData(data: RawCSV[]): PreparedRow[] {
+  const toBigInt = (s: string) => new BigNum(s.replace(/,/g, ""))
```

Comment:
The short title seemed a bit obvious to me, perhaps we should discard such short titles if possible